scripts/jenkins/Jenkinsfile

```diff
@@ -773,24 +817,50 @@ def xunitUpload() {
     ])
 }
 
+/* Create a new Docker Image Description
+ *
+ * @param name Name of the image, will be extended with registry, a common
+ *             prefix and a tag
+ * @param tagFun Closure describing how the tag should be formatted
+ * @param context Context for the docker build
```

Comment:
What is a context in this context?

Reply:
https://docs.docker.com/engine/reference/builder/#usage

Reply:
The wording is related to docker. I expect people to kind of know what Docker is before manipulating docker images in the build server configuration.

Reply:
Even within docker it is called "build context". The Jenkinsfile Docu is in many places so sloppy (e.g. with its terminology) that it is not useful that way. Please do not discuss every word but make it more clear. The Docker docu you refer to is also sloppy (with its terminology) but they have at least many examples (which the Jenkinsfile does not have at all).

And please reduce the assumptions of what people know. You should at least state what you assume (e.g. in BUILDSERVER.md).

Reply:
Btw. I know you can do better. Please do not try to prove a point here :wink: 

If the docu somewhere else in Elektra is not according to this standard, please create issues.

Reply:
> Please do not try to prove a point here

Not trying anything. Just getting tired of working on it. Pretty much haven't done except wrestle with build server stuff for the past two months full time.

Reply:
Unfortunately it is not really transparent what you are doing unless you make it visible (especially the changes on the build server where there is no changelog at all), which usually means you need to document it somehow.